### PR TITLE
Change http to https for security links

### PR DESCRIPTION
**Pull request by trungnvfet**
Signed-off-by: Nguyen Van Trung <trungnv@vn.fujitsu.com>

**What this PR does / why we need it**:
For security, Should change into https links.
**Special notes for your reviewer**:
NONE
**If applicable**:
[x this PR contains documentation

### CODE REVIEW

File: docs/related.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ or [pull request](https://github.com/helm/helm/pulls).
 ## Article, Blogs, How-Tos, and Extra Documentation
 
 - [Awesome Helm](https://github.com/cdwv/awesome-helm) - List of awesome Helm resources
-- [CI/CD with Kubernetes, Helm & Wercker ](http://www.slideshare.net/Diacode/cicd-with-kubernetes-helm-wercker-madscalability)
-- [Creating a Helm Plugin in 3 Steps](http://technosophos.com/2017/03/21/creating-a-helm-plugin.html)
-- [Deploying Kubernetes Applications with Helm](http://cloudacademy.com/blog/deploying-kubernetes-applications-with-helm/)
+- [CI/CD with Kubernetes, Helm & Wercker ](https://www.slideshare.net/Diacode/cicd-with-kubernetes-helm-wercker-madscalability)
+- [Creating a Helm Plugin in 3 Steps](https://technosophos.com/2017/03/21/creating-a-helm-plugin.html)
```
Comment (mattfarina) [inline docs/related.md]: Unortunately, this link does not work for `https`. Can you revert it to `http`

File: docs/related.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ Tools layered on top of Helm or Tiller.
 - [Chartify](https://github.com/appscode/chartify) - Generate Helm charts from existing Kubernetes resources.
 - [Codefresh](https://codefresh.io) - Kubernetes native CI/CD and management platform with UI dashboards for managing Helm charts and releases
 - [Cog](https://github.com/ohaiwalt/cog-helm) - Helm chart to deploy Cog on Kubernetes
-- [Drone.io Helm Plugin](http://plugins.drone.io/ipedrazas/drone-helm/) - Run Helm inside of the Drone CI/CD system
+- [Drone.io Helm Plugin](https://plugins.drone.io/ipedrazas/drone-helm/) - Run Helm inside of the Drone CI/CD system
```
Comment (mattfarina) [inline docs/related.md]: Another link that does not appear to support `https`.  Can you please revert to `http`

**Review by mattfarina — COMMENTED** [review]
A couple of the links are not working with https. Can you please fix these

**Comment by trungnvfet** [comment]
@mattfarina Thank you for your review. I just updated new PR.

**Review by mattfarina — APPROVED** [review]

**Pull request merged**